recipes/apriltag/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.16)
+project(apriltag_ros)
```

Comment:
```suggestion
project(apriltag_ros LANGUAGES C)
```

Reply:
I was looking at the wrong CMakeLists.txt. I thought the error came from the apriltag library, where `LANGUAGES C` is set like this. But now I see that the error comes from the compilation of the test source file. Thanks for pointing this out.